wcfsetup/install/files/lib/system/WCF.class.php

```diff
@@ -590,8 +593,8 @@ protected function initApplications() {
 				$application->__run();
 			}
 			
-			// refresh the session 1 minute before it expires
-			self::getTPL()->assign('__sessionKeepAlive', SESSION_TIMEOUT - 60);
+			/** @deprecated The below variable is deprecated. */
+			self::getTPL()->assign('__sessionKeepAlive', 60 * 50);
```

Comment:
```suggestion
			self::getTPL()->assign('__sessionKeepAlive', 59 * 60);
```

This matches the previous default value for consistency. Also changed the order to represent `<minutes> * <seconds>`.